src/pages/Search/__test__/Search.test.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Route, MemoryRouter } from 'react-router-dom';
+import {
+  render,
+  cleanup,
+  waitForElementToBeRemoved,
+  fireEvent,
+  waitFor,
+} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { ThemeProvider } from 'styled-components';
+import axiosMock from 'axios';
+import Search from '../index';
+import Theme from '../../../styles/theme';
+import posts from './posts.json';
+
+jest.mock('axios');
+afterEach(cleanup);
+
+const renderWithRouter = (component, subreddit) => ({
+  ...render(
```

Comment:
This spread operator is not necessary. Forgot about that a couple of times already. This is enough

```
const renderWithRouter = (component, subreddit) => render(
```